apis/v1alpha1/httproute_types.go

```diff
@@ -167,6 +167,20 @@ type HTTPRouteRule struct {
 	// path match on "/", which has the effect of matching every
 	// HTTP request.
 	//
+	//
+	// Although a client request may technically match multiple HTTP route rules,
+	// only one rule may ultimately receive the request. Matching precedence
+	// MUST be determined in order of the following criteria:
+	//
+	// * The most specific match is determined by the longest matching combination
+	//   of hostname and path. In the event of a tie, the larger number of header
```

Comment:
Nit: The second sentence could potentially be a second bullet point.

Reply:
> larger number of header

What does this exactly mean?